rust/ql/test/query-tests/unusedentities/main.rs

```diff
@@ -302,6 +350,27 @@ fn if_lets_matches() {
         p => { // BAD: unused variable
         }
     }
+
+    let duration1 = std::time::Duration::new(10, 0); // ten seconds
+    assert_eq!(duration1.as_secs(), 10);
+
+    let duration2:Result<std::time::Duration, String> =
+        Ok(std::time::Duration::new(10, 0));
+    match (duration2) {
+        Ok(n) => { println!("duration was {} seconds", n.as_secs()); }
+        Err(_) => { println!("failed"); }
+    }
+
+    let(left,
```

Comment:
Maybe a space between `let` and `(left,`?

Reply:
Also done.